pkg/security/secl/model/model.go

```diff
@@ -638,3 +639,13 @@ type MProtectEvent struct {
 	VMProtection  int    `field:"vm_protection"`
 	ReqProtection int    `field:"req_protection"`
 }
+
+// SignalEvent represents a signal event
+type SignalEvent struct {
+	SyscallEvent
+
+	Type                    uint32             `field:"type"`   // Signal type (ex: SIGHUP, SIGINT, SIGQUIT, etc)
+	PID                     uint32             `field:"pid"`    // Target PID
+	Target                  ProcessContext     `field:"target"` // Target process context
+	TargetProcessCacheEntry *ProcessCacheEntry `field:"-"`
```

Comment:
I know this pattern comes from the PTraceEvent but I'm wondering why we need to have twice the same info, `Target` with a `ProcessContext` would be enough ?

Reply:
To be able to make the proposed change (for both PTrace and Signal), a rework of `newProcessContextSerializer` should be done first.